docs/DeveloperGuide.md

```diff
@@ -124,7 +124,7 @@ The nontrivial methods in `ExpenseList` class contain some sort of exception han
 
 The following diagram is a UML class diagram for `Expense` and `ExpenseList`:
 
-![ExpenseAndExpenseList.png](/docs/diagrams/images/ExpenseAndExpenseList.png "UML Class Diagram for Expense and ExpenseList")
```

Comment:
Issue with the previous one is that it is using an absolute path and not a relative path. This doesn't really work well with Github Pages